packages/medusa-react/src/hooks/store/payment-collections/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./queries"
+export * from './mutations'
```

Comment:
todo: Same as other comment
```suggestion
export * from "./mutations"
```